packages/driver/cypress/integration/commands/request_spec.js

```diff
@@ -830,7 +830,7 @@ describe('src/cy/commands/request', () => {
           expect(this.logs.length).to.eq(1)
           expect(lastLog.get('error')).to.eq(err)
           expect(lastLog.get('state')).to.eq('failed')
-          expect(err.message).to.eq('`cy.request()` must be provided a fully qualified `url` - one that begins with `http`. By default `cy.request()` will use either the current window\'s origin or the `baseUrl` in `cypress.json`. Neither of those values were present.')
+          expect(err.message).to.eq('`cy.request()` must be provided a fully qualified `url` - one that begins with `http`. By default `cy.request()` will use either the current window\'s origin or the `baseUrl` in `cypress.config.js`. Neither of those values were present.')
```

Comment:
This error is slightly inaccurate since the baseUrl could have been passed via the --config outside of the configuration file. But, I know this wasn't introduced in this PR. 😝

Reply:
I fixed the error message to say option instead of the file, since it can com from either configfile or the CLI options but is most likely something wrong within the test.